src/create-glamorous.js

```diff
@@ -7,11 +7,17 @@ import {PropTypes} from './react-compat'
 import withTheme from './with-theme'
 import getGlamorClassName from './get-glamor-classname'
 
+let glamorousInstance = 1
+
 export default createGlamorous
 
 function createGlamorous(splitProps) {
   // TODO: in a breaking version, make this default to true
-  glamorous.config = {useDisplayNameInClassName: false}
+  glamorous.config = {
+    useDisplayNameInClassName: false,
+    idCounter: 1,
+    instanceId: glamorousInstance++,
```

Comment:
Good point. There should probably be a rehydration strategy for the glamorous instance id and the component id... There are _very_ few cases where there will be more than one instance of glamorous (you'd have to create your own which very few (no?) people do). But the glamorous component ID should probably be rehydrated properly... 🤔 

I have no idea how to do this. Or if I'm even understanding the problem properly, or even if it is indeed a problem (which I think it is).